.github/workflows/publish.yml

```diff
@@ -82,6 +82,7 @@ jobs:
         run: |
           make clean
           docker system prune -f -a
+          docker rm -f $(docker ps -aq) && docker volume rm -f $(docker volume ls -q)
```

Comment:
It looks like we already do not care about local cachees between different steps here... So should be fine, I guess.

Reply:
If you do `docker system prune --volumes` does it not clean those up?

Reply:
`docker volume rm -f $(docker volume ls -q)` forcibly deletes all volumes regardless of usage, while `docker system prune --volumes` safely removes only unused (dangling) volumes.